scipy/interpolate/_cubic.py

```diff
@@ -164,15 +164,16 @@ class PchipInterpolator(CubicHermiteSpline):
 
     Parameters
     ----------
-    x : ndarray
+    x : ndarray, shape (npoints, )
         A 1-D array of monotonically increasing real values. ``x`` cannot
         include duplicate values (otherwise f is overspecified)
-    y : ndarray
-        A 1-D array of real values. ``y``'s length along the interpolation
+    y : ndarray, shape (..., npoints, ...)
+        A 1-D array of real values. ``y``'s length along the *first*
         axis must be equal to the length of ``x``. If N-D array, use ``axis``
         parameter to select correct axis.
     axis : int, optional
-        Axis in the y array corresponding to the x-coordinate values.
+        Axis in the y array corresponding to the x-coordinate values. Interpolation
+        defaults to the first axis of ``y``.
```

Comment:
Nit: add default is zero or default is `axis=0` or some such, so that it stands out on a quick scan.

Reply:
Done